boot/script-loader/mruby-lvgui-native/lvgui.nix

```diff
@@ -97,12 +99,13 @@ in
     pname = "lvgui";
     version = "2021-07-20";
 
-    src = fetchFromGitHub {
-      repo = "lvgui";
-      owner = "mobile-nixos";
-      rev = "ba381a2749848ac69e08e9e59fe8a14054bcb24e";
-      sha256 = "0989m73kcrybgadszk1ffh4cbcfrmi0f79lbg87wrn5bxwaqbawr";
-    };
+    src = /Users/samuel/Projects/mobile-nixos/projects/lvgui;
```

Comment:
This local path looks like a forgotten debug test, right?

Reply:
Oh no

Reply:
Wel... it worked on my machine :/.